### PR TITLE
Remove authentication check from auth/callback route [INT-223]

### PR DESCRIPTION
**Pull request by ckoberlein**
## Ticket
https://dronedeploy.atlassian.net/browse/INT-223

## Work Done
- Add `/auth/callback` to `function-wrapper` configuration for ignoring auth
- Add new `/token` route to actually store the access token to datastore
- Remove code related to the state parameter

By ignoring the JWT check on the /auth/callback (accessed by oauth provider), we simplify everything, but it does require separate call from UI app client side to store the token.

### CODE REVIEW

File: handler.js [hunk 1]
```diff
@@ -128,27 +133,17 @@ const storeTokenData = (table, username, tokenData, res, isRefresh) => {
       throw new Error(JSON.stringify(rowData.errors[0]));
     }
 
-    if (isRefresh) {
```
Comment (jryoo) [inline handler.js]: why don't we need this anymore?

File: handler.js [hunk 2]
```diff
@@ -94,7 +97,9 @@ const packageError = (err) => {
 };
 
 const isEmptyToken = (data) => {
-  return data.accessToken === "" || data.refreshToken === "";
+  const isUndefined = (!data.accessToken || !data.refreshToken);
```
Comment (mikedeploysdrones) [inline handler.js]: Isn't this actually isUndefinedOrEmpty since !"" will also be true here?

File: handler.js (hunk 1 above)
Comment (ckoberlein) [reply handler.js]: We are now making a separate call to store the token data from the UI --- meaning that during the initial auth flow when we store the token, it's simply an API call and does not require an HTML response.  This is how the refresh flow works so there's no need to have the conditional.

File: handler.js (hunk 2 above)
Comment (ckoberlein) [reply handler.js]: @mikedeploysdrones Fair point - was seeing some odd behavior in my testing due to a bug I introduced, so I added the extra check to be certain.  The data didn't even have the properties, so undefined wasn't === "" and was causing inaccurate behavior.

**Pull request merged**